CONTRIBUTING.md

```diff
@@ -1,10 +1,4 @@
-# How to contribute
-
-Thank you for contributing to this project.
-
-We welcome bug reports and discussions about new function in the issue tracker, and we also welcome proposed new features or bug fixes via pull requests.
-
-You should read these guidelines to help you contribute.
+# Developer guide
```

Comment:
I think I will rename this file to dev guide at some point, based on a conversation with Demetri

Reply:
Great, was thinking about that too! I'll update the link after it's renamed. @stewartfrancis